package.json

```diff
@@ -92,6 +94,7 @@
     "c8": "^8.0.1",
     "chai": "^4.3.10",
     "chai-http": "^4.4.0",
+    "cross-env": "^7.0.3",
```

Comment:
If this is in `dependencies` already, we shouldn’t also need it in `devDependencies`.

```suggestion
```